### PR TITLE
Change studio url from https to http

### PR DESCRIPTION
**Pull request by lyw07**
# Details

### Summary

* description of the change
* manual verification steps performed
* screenshots if the PR affects the UI

### Reviewer guidance

description of how to test the changes

### References

when applicable, please provide:

* references to related issues and PRs
* links to mockups or specs for new features
* links to the diffs for dependency updates, e.g. in iceqube or the perseus plugin

# Contributor Checklist

- [ ] PR has the correct target milestone
- [ ] PR has the appropriate labels
- [ ] Changes in the PR do not introduce accessibility regressions ([confimed by testing with one of the recommended tools](http://kolibri.readthedocs.io/en/develop/dev/manual_testing.html#accessibility-a11y-testing)) 
- [ ] If PR is ready for review, it has been assigned or requests review from someone
- [ ] Documentation is updated as necessary
- [ ] External dependency files are updated (`yarn` and `pip`)
- [ ] If internal dependency is updated, link to diff is included
- [ ] Screenshots of any front-end changes are in the PR description
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] You've added yourself to AUTHORS.rst if you're not there

# Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually

### CODE REVIEW

**Comment by benjaoming** [comment]
What's this PR about?

Also, isn't it a regression to revert from HTTPS to HTTP?

And why isn't there a new issue created to deal with this?

File: kolibri/deployment/default/settings/base.py [hunk 1]
```diff
@@ -139,7 +139,7 @@
     os.makedirs(CONTENT_STORAGE_DIR)
 
 # Base default URL for downloading content from an online server
-CENTRAL_CONTENT_DOWNLOAD_BASE_URL = "https://studio.learningequality.org"
+CENTRAL_CONTENT_DOWNLOAD_BASE_URL = "http://studio.learningequality.org"
```
Comment (benjaoming) [inline kolibri/deployment/default/settings/base.py:142]: I think it's fine to download contents over the open internet -- but how do we ensure that there are no secrets shared via this URL?

:+1: for the naming of the URL, it does imply that it's only intended for content download... but might suffix it with `_INSECURE`, to be.... secure :tada:

**Comment by indirectlylit** [comment]
created a follow-up issue #2685.

see also:

* #2324
* #2544

**Comment by lyw07** [comment]
@benjaoming Sorry I forgot to put it the description after testing the pex file, this PR was to fix #2544

**Comment by benjaoming** [comment]
Thanks for creating the follow-up issue @indirectlylit !

**Pull request merged**